src/compiler.js

```diff
@@ -391,10 +391,10 @@ class SizeOfCompiler extends Compiler {
    * @param {*} fn
    */
   addNativeType (type, fn) {
+    this.primitiveTypes[type] = `native.${type}`
     if (!isNaN(fn)) {
-      this.primitiveTypes[type] = fn
+      this.native[type] = (value) => { return fn }
```

Comment:
Can we inline native functions?

Reply:
In this case fn is a number (`!isNaN(fn)`) It cannot be inline because it needs to have the same function interface, but anyway the JIT compiler will optimise it for us.